docs/questions/start.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Questions overview
+---
+
+# Questions
+
+Questions are queries plus their visualization. You can ask questions using Metabase's graphical query builder, or create a native/SQL query.
+
+## Query builder
+
+### [Asking questions](./query-builder/introduction.md)
+
+An introduction to Metabase's graphical query builder.
+
+### [Custom expressions](./query-builder/expressions.md)
+
+Custom expressions, like spreadsheet formulas, are tools you can use in the graphical query builder to ask more complicated questions.
+
+### [List of expressions](./query-builder/expressions.md)
+
+A list of the aggregations and functions available in the graphical query builder.
+
+### [Joining data](./query-builder/join.md)
+
+You can join data to combine your current data with another table, or even with a saved question.
+
+## Native query editor
+
+Also known as the SQL editor (we say native because you can also query data sources that don't use SQL, like MongoDB and Google Analytics).
+
+### [Introduction to the native query editor](./native-editor/writing-sql.md)
+
+Write native code to query your data source.
```

Comment:
```suggestion
Write native code to query your database.
```

Let's avoid "data source" when possible, because people often make leaps like: Excel = data source = database 😅

Reply:
Yeah, it's one of those native/sql things, because of Google Analytics (which is a data source, not a database). But I'm fine with just saying database.